feature/community/src/main/java/com/iteneum/community/presentation/CommunityComposable.kt

```diff
@@ -0,0 +1,60 @@
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.iteneum.community.R
+import com.iteneum.community.data.CommunityViewModel
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.designsystem.components.LpIconTextCard
+import com.iteneum.designsystem.theme.LeasePertTheme
+
+@Composable
+fun CommunityUI(viewModel: CommunityViewModel = viewModel()) {
+    val width = LeasePertTheme.sizes.extraSize124
+    val height = LeasePertTheme.sizes.extraSize128
+    val padding10 = LeasePertTheme.sizes.midSmallSize
+    val padding20 = LeasePertTheme.sizes.minorRegularSize
+
+    val cards = remember { viewModel.buttonCards }
+    Column(modifier = Modifier.fillMaxWidth()) {
```

Comment:
Are ou using this Column? because I don't see any components inside

Reply:
it is used to prevent title to go behind the list of community buttons

Reply:
basically all the screen is under that column 
![imagen](https://user-images.githubusercontent.com/128720218/232522694-921cce26-3865-493c-884e-513a96e59ce4.png)